toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -361,3 +367,24 @@ func loadOrDisableModules(modules imagecustomizerapi.Modules, imageChroot *safec
 
 	return nil
 }
+
+func addCustomizerRelease(imageChroot *safechroot.Chroot) error {
+	var err error
+
+	logger.Log.Infof("Creating image customizer release file")
+	micVersion := "0.0.0"
+	currentTime := time.Now().Format("2023-10-24T23:30:45Z")
+
+	customizerreleaseFilePath := filepath.Join(imageChroot.RootDir(), "/etc/mariner-customizer-release")
+	err = file.Write(fmt.Sprintf("%s=\"%s\"", "VERSION", micVersion), customizerreleaseFilePath)
+	if err != nil {
+		return fmt.Errorf("error writing version to '%s': %w", customizerreleaseFilePath, err)
+	}
+
+	err = file.Write(fmt.Sprintf("%s=\"%s\"", "BUILD_DATE", currentTime), customizerreleaseFilePath)
```

Comment:
I believe calling `file.Write` overwrites any existing file. So, calling `file.Write` twice won't work.

Reply:
It makes sense to aggregate and dump things in file once.

Reply:
sorry overlooked this, will append them